src/slim-select/helpers.ts

```diff
@@ -1,9 +1,21 @@
-// Generate an 8 character random string
```

Comment:
Part of why i use typescript is because I shouldnt have to set a bunch of comments describing params. Lets just keep them as simple description comments and let the typescript explain itself.

Reply:
Sounds good, will do